src/routes.js

```diff
@@ -12,13 +12,14 @@ import NotFound from './views/NotFound';
 
 const page = View => () => <Layout><View /></Layout>;
 
-const entryPage = View => () => <LayoutEntry><View /></LayoutEntry>;
+// TODO: this call gives react warning about "unique key propety"
+const entryPage = components => () => <LayoutEntry>{components}</LayoutEntry>;
 
 // Routes for redux-router-kit
 const routes = {
   '/': page(QuotedStartups),
   '/test': page(Test),
-  '/login': entryPage(Login),
+  '/login': entryPage([<Login key="12" />, <div>test</div>, 'test']),
```

Comment:
I don't think it makes sense to define the columns in the routing file. There should simply be a `entryPage(LoginView)`. How the `LoginView` looks (layout, columns, etc) is up to the implementation.